bin/GameIndex.yaml

```diff
@@ -3354,6 +3354,8 @@ SCPS-15056:
   region: "NTSC-J"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - EETimingHack # Fixes DMA errors 
```

Comment:
This timing hack needs to be removed.

Reply:
Have you played through the Japanese version to make sure it isn't required?